### PR TITLE
Flex box styling

### PR DESCRIPTION
**Pull request by connorhoman**
Added: styling (flexbox, icons, spacing, font/color), click handling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
             4) psql listing_info (Use DB)
             5) \i /Users/connorhoman/Desktop/listinginfo/database/db_create.sql {substitute w/ your file path} (Create Table)
             6) \q (To exit psql shell)
-            7) NPM run seed
```
Comment (jeffday90) [inline README.md:28]: lol

File: dist/Components/lowerInfo/cancellation.jsx [hunk 2]
```diff
@@ -31,6 +31,12 @@ const Button = styled.div`
     text-decoration:underline;
   }
 `;
+const Line = styled.hr`
+  color: #E6E4E4;
+  height: 1px;
+  background-color: #E6E4E4;
```
Comment (jeffday90) [inline dist/Components/lowerInfo/cancellation.jsx:37]: stoked you are getting with the styled components program!

File: package.json [hunk 3]
```diff
@@ -18,6 +18,9 @@
     "*.css": "stylelint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.18",
```
Comment (jeffday90) [inline package.json:21]: fortawesome?

**Review by jeffday90 — APPROVED** [review]
is fort awesome where you are getting your icons? I need to figure out a place to get some of these dang old arrows.

File: package.json (hunk 3 above)
Comment (connorhoman) [reply package.json:21]: Its really Font Awesome haha, not sure why thats their install.  But I just followed these instructions and it was pretty easy: 
https://fontawesome.com/how-to-use/on-the-web/using-with/react

**Pull request merged**